### DISCUSSION TITLE
Hey everyone the newbie of all newbies is here. i promise that by the time im done here. im gonna really blow some minds. Mine included.

### DISCUSSION THREAD

**Discussion by devito420** [community/community#108418] [New to GitHub]
hey GitHub, long time reader finally got the balls to out myself. First off let me say this, This site along with everything going on now is freaking amazing. I truly believe we are living in potentially the end time (i not super religious). But with every ending comes a new brighter day. Thats what im focusing on. So anyway names Devito. you know like the big twin in the movie twins..lol

**Comment by sapalazz** [comment]
Hey @devito420, welcome to the club. I'm quite a newbie myself, but happy to browse discussions, read people's struggles & adventures and learn all I can from you. Best wishes 🔆

**Comment by queenofcorgis** [comment]
👋 Welcome to the GitHub Community on Discussions, @devito420  Happy to have you here and thanks for the kinds words 🙂

To help keep our space clutter-free, we recommend introducing yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204) and checking out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776). This post will be closed and locked to keep our community nice and tidy 🧹.
